services/net/image/ImageAction.cs

```diff
@@ -154,18 +156,35 @@ public ImageAction(IApiService api, IOptions<ImageOptions> options, ILogger<Imag
                     var reference = await this.FindContentReferenceAsync(manager.Ingest.Source?.Code, newReference.Uid);
                     if (reference == null)
                     {
+                        isNewSourceContent = true;
                         reference = await this.Api.AddContentReferenceAsync(newReference);
+                        Logger.LogDebug($"Image Ingest : ADD NEW : [{manager.Ingest.IngestType?.Name ?? "Image"} - {manager.Ingest.Name}] : {path}");
+                    } else {
+                        // if we already have a contentReference, check if this is actually an update to the image
+                        if (newReference.PublishedOn > reference.PublishedOn)
+                        {
+                            isUpdatedSourceContent = true;
+                            // update the existing refernce published on date with the new date
+                            reference.PublishedOn = newReference.PublishedOn;
+
+                            // What about the worst case scenario: one Editor changes it in MMI and another Editor changes it in TNO?
```

Comment:
I don't think we care about this scenario.  In TNO they don't even look at front pages in the editor app.

Reply:
that comment was acopy/paste from lesewhere.  not relevant